server/src/models/BoardGame.js

```diff
@@ -12,32 +12,43 @@ class BoardGame extends unique(Model) {
   }
 
   static get relationMappings() {
-	const { Review } = require("./index")
-
-	return {
-		reviews: {
-			relation: Model.HasManyRelation,
-			modelClass: Review,
-			join: {
-				from: "boardGames.id",
-				to: "reviews.boardGameId"
-			}
+		const { Review } = require("./index")
+		const {User} = require("./index")
+
+
+		return {
+			reviews: {
+				relation: Model.HasManyRelation,
+				modelClass: Review,
+				join: {
+					from: "boardGames.id",
+					to: "reviews.boardGameId"
+				}
+			}, 
+			
+			user: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: User,
+        join: {
+          from: "boardGame.userId",
+          to: "users.id"
+        }
+      }
```

Comment:
THIS...MAKES....NICK....SAD/MAD.....SMAD
fix ze indentation!! 

https://giphy.com/clips/cameronpaulsmith-sesame-street-elmo-fire-ydP26cNviWHKHcPeAp